firstvoices/backend/views/custom_order_recalculate_views.py

```diff
@@ -0,0 +1,53 @@
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from backend.models import CustomOrderRecalculationPreviewResult
+from backend.serializers.async_results_serializers import (
+    CustomOrderRecalculationPreviewResultDetailSerializer,
+)
+from backend.tasks.alphabet_tasks import recalculate_custom_order_preview
+from backend.views.base_views import FVPermissionViewSetMixin, SiteContentViewSetMixin
+from backend.views.exceptions import CeleryError
+
+
+class CustomOrderRecalculatePreviewView(
+    FVPermissionViewSetMixin, SiteContentViewSetMixin, ModelViewSet
```

Comment:
I noticed that the endpoint `/api/1.0/sites/{site_slug}/dictionary-cleanup/preview/{id}/` is still enabled and shown in the `/api/docs/` page. It looks like this endpoint is not needed and could be removed. An easy way to do that is to subclass the view with the needed subset of mixins in `ModelViewSet` (I believe it is `mixins.RetrieveModelMixin` that adds the `/{id}/` endpoint).

Reply:
Nice catch! Made a small change to disable the detail view